makefile

```diff
@@ -52,7 +52,7 @@ ifeq ($(ENABLE_TRUSTZONE),1)
 endif
 
 INCLUDES := -I$(BUILDDIR) -Iinclude
-CFLAGS := -O2 -g -fno-builtin -finline -Wno-multichar -Wno-unused-parameter -Wno-unused-function -include $(CONFIGHEADER)
+CFLAGS := -fcommon -O2 -g -fno-builtin -finline -Wno-multichar -Wno-unused-parameter -Wno-unused-function -include $(CONFIGHEADER)
```

Comment:
`-fcommon` seems to be indeed working with gcc 10, good find! (Note: In general it looks like `-fcommon` isn't really something you want, but this is all ugly legacy code from qcom so it's really the best solution for now...)

Can you put it on an extra line like the `-Werror` below and add a short comment that it's to fix build on gcc 10?